From nobody Mon Sep 17 00:00:00 2001
From: David Hollis <dhollis@davehollis.com>
Date: Thu Jan 5 14:39:49 2006 -0500
Subject: [PATCH] USB: asix - Add device IDs for 0G0 Cable Ethernet
Patch-mainline: 2.6.16-rc3

Add device IDs for the 0G0 Cable Ethernet device as reported by
Charles Lepple <clepple@gmail.com>.

Signed-off-by: David Hollis <dhollis@davehollis.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---

 drivers/usb/net/asix.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

e955dacf2d177f877e930816627b98e1faf2f45c
diff --git a/drivers/usb/net/asix.c b/drivers/usb/net/asix.c
index 5411816..3094970 100644
--- a/drivers/usb/net/asix.c
+++ b/drivers/usb/net/asix.c
@@ -916,6 +916,10 @@ static const struct usb_device_id	produc
 	// Linksys USB200M Rev 2
 	USB_DEVICE (0x13b1, 0x0018),
 	.driver_info = (unsigned long) &ax88772_info,
+}, {
+	// 0Q0 cable ethernet
+	USB_DEVICE (0x1557, 0x7720),
+	.driver_info = (unsigned long) &ax88772_info,
 },
 	{ },		// END
 };
-- 
1.1.4.gdcc6

